### DISCUSSION TITLE
Story is not showing icon importing from ion-icon

### DISCUSSION THREAD

**Discussion by rajanlagah** [storybookjs/storybook#13994] [Help]
**Describe the bug**
I was using icon from [here](https://ionicons.com/). It is showing icon in React app but it is not showing that same icon in story. I then click to inspect element and in html I can see the ion-icon's icon

**To Reproduce**
- Create any story 
- Add CDN from [here](https://ionicons.com/usage) 
- Choose any icon from [here](https://ionicons.com/) 
- Import it in any React component 
- Write its story
- You will see icon is not appearing 

**Expected behavior**
Icon should appear just like it appear in main application 

**Screenshots**
![Screenshot from 2021-02-04 23-48-18](https://user-images.githubusercontent.com/20952569/106937840-3d5eeb80-6744-11eb-9354-0ca496c92b1c.png)

**Code snippets**
If applicable, add code samples to help explain your problem.
No code needed i guess 

**System**
Please paste the results of `npx sb@next info` here.
```
Environment Info:

  System:
    OS: Linux 5.8 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (24) x64 AMD Ryzen 9 3900X 12-Core Processor
  Binaries:
    Node: 14.15.4 - /usr/local/bin/node
    Yarn: 1.22.5 - ~/.yarn/bin/yarn
    npm: 6.14.10 - /usr/local/bin/npm
  Browsers:
    Chrome: 88.0.4324.96
    Firefox: 85.0


```
**Additional context**
I can totally help solving problem

**Comment by RyanClementsHax** [comment]
Hopefully this comment helps you out https://github.com/ionic-team/ionicons/issues/670#issuecomment-848864689
